packages/core/src/components/ModalNew/layouts/ModalMedia.module.scss

```diff
@@ -1,8 +1,17 @@
 .media {
-  align-items: normal;
-  align-self: normal;
-  width: 100%;
-  height: auto;
+  max-width: 100%;
   max-height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-shrink: 0;
   overflow: hidden;
+  position: relative;
+
+  > * {
```

Comment:
This is usually bad for performance as css selectors are read from right to left. Any way to avoid it?

Reply:
I can just target img and video tags instead

Reply:
I'm not sure how Lottie would behave anyway with the current CSS I applied

Reply:
We should probably have Lottie as a dev dep for this kind of cases

Reply:
Dev dep? Why? Plus I feel like that this css is creating a side effect for the users, don't you think? Because they can provide an img element and use some css on that element but you will most likely override their css

Reply:
Their css should "win"
I'm using 0,1,0 specificity 
so if they override it, their css would be the one rendered
also, you suggested in a different PR to apply a predefined css
otherwise the ModalMedia is redundant as ModalContent

Reply:
I meant predefined css for an image that you provided by a prop, not by children, but this was dropped, no? 
But anyway, I mean that targeting elements inside your component is creating a side effect. I don't think that the css is bad, it just should only affect the elements inside your component and not elements provided by the user. Plus you're applying style that may not fit to all elements like object-fit. I think the best combination would be to have an Image/Video components or maybe Media. This along with the ModalMedia will get along well. But probably out of this scope

Reply:
I need to release the layouts with a solution for that
I cannot leave it with every user doing whatever they want, it would create inconsistency
let's have a talk about it tomorrow